### PR TITLE
Homework6: Valeria Kudelko

### PR DESCRIPTION
**Pull request by ValeriaKudelko**

### CODE REVIEW

File: firstfile.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
+"""Writing Python Code."""
+
+# !1.Заменить символ “#” на символ “/” в строке 'www.my_site.com#about'.
+Sq = 'www.my_site.com#about'.replace('#', '/')
```
Comment (Stml89) [inline firstfile.py]: please read this PEP - https://peps.python.org/pep-0008/#descriptive-naming-styles

File: secondfile.py [hunk 2]
```diff
@@ -0,0 +1,44 @@
+"""Writing Python Code."""
+
+# 1.Перевести строку в список "Robin Singh" => ["Robin”, “Singh"].
+zu1 = []
```
Comment (Stml89) [inline secondfile.py]: you do not need empty list creation

**Review by Stml89 — CHANGES_REQUESTED** [review]
Stiil an issue with pylint, please fix it

File: secondfile.py (hunk 2 above)
Comment (ValeriaKudelko) [reply secondfile.py]: corrected

**Review by Stml89 — APPROVED** [review]

**Pull request merged**